.github/workflows/linux_meson.yml

```diff
@@ -198,3 +200,55 @@ jobs:
           cd doc
           python3-dbg -m pip install pytest pytest-xdist pytest-timeout mpmath gmpy2 threadpoolctl pooch
           python3-dbg -m pytest --pyargs scipy -n2 --durations=10 -m "not slow"
+
+  #################################################################################
+  gcc8:
+    # Purpose is to examine builds with oldest-supported gcc.
+    name: Build with gcc-8
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-20.04  # 22.04 doesn't support gcc-8
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+
+      - name: Setup OS
+        run: |
+          sudo apt-get -y update
+          sudo apt install -y g++-8 gcc-8 gfortran-8
+          sudo apt install -y libatlas-base-dev liblapack-dev libgmp-dev \
+            libmpfr-dev libmpc-dev pkg-config libsuitesparse-dev
+
+          # install openblas
```

Comment:
It's not necessary to both install ATLAS above and then OpenBLAS here. I suggest using ATLAS and removing these lines (see the `python-debug` job for how to use ATLAS).

Reply:
Do we have page explaining these? AFAIK we have this https://scipy.github.io/devdocs/dev/contributor/building.html#system-level

But to me it's not clear what is what and there is no mention of `libgmp-dev libmpfr-dev libmpc-dev pkg-config libsuitesparse-dev`

Reply:
They're very very indirect optional dependencies, for `gmpy2` which makes `mpmath` (used in `scipy.special` testing) faster, and to build `scikit-umfpack`. The latter just broke, I'm having a look at fixing that.

Reply:
I don't really want to explain that in our docs, just a code comment will do.